Automattic/wp-calypso
# Stepper: Change unfocused items blur value to 3px

#### Proposed Changes

* Title says it all.

#### Testing Instructions
1. Go to /setup/patterns/?flow=link-in-bio.
2. Unfocused items should have 3px blur.
